terraform-unity/variables.tf

```diff
@@ -33,7 +33,7 @@ variable "counter" {
 variable "release" {
   description = "The software release version."
   type        = string
-  default     = "2.0.1"
+  default     = "2.0.2-beta.0"
```

Comment:
@LucaCinquini Should we remove the -beta.0 before cutting the release?

Reply:
Yes - thanks for catching that!